packages/devtools_app/lib/src/timeline/flutter/timeline_model.dart

```diff
@@ -0,0 +1,1099 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:collection';
+import 'dart:math' as math;
+
+import 'package:meta/meta.dart';
+
+import '../../profiler/cpu_profile_model.dart';
+import '../../service_manager.dart';
+import '../../trees.dart';
+import '../../utils.dart';
+import 'timeline_controller.dart';
+
+class TimelineData {
```

Comment:
in this file I combined the classes  `FrameBasedTimelineData` and `FullTimelineData` into `TimelineData` (which was previously the abstract superclass of both).